lmnet/tests/lmnet_tests/networks_tests/test_quantize_param_init.py

```diff
@@ -0,0 +1,115 @@
+import tensorflow as tf
```

Comment:
If you use `reset_default_graph`,  you can implement test more simple :+1: 
https://github.com/blue-oil/blueoil/blob/acb5a270f201f34fe5a5b27a4b395d9c3a838b27/lmnet/tests/conftest.py#L23

Reply:
As far as I know, fixture is used to setup the test automatically.
The smallest scope that fixture can be called is function [REF](https://docs.pytest.org/en/latest/fixture.html), which means it can be called before function is called.

However, I think `tf.reset_default_graph()` must be called every time before we create the network so I think using `scope=function` for fixture may not be enough in this test?

Reply:
you can divide test function.
Ideally, test function should do one test.

Reply:
I divided each test case into different functions and use the `reset_default_graph` fixture.